source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+.. command-block::
+
+   qiime feature-table summarize \
+     --i-table table.qza \
+     --o-visualization table.qzv \
+     --m-sample-metadata-file sample-metadata.tsv
+   qiime feature-table tabulate-seqs \
+     --i-data rep-seqs.qza \
+     --o-visualization rep-seqs.qzv
+
+Generate a tree for phylogenetic diversity analyses
+---------------------------------------------------
+
+QIIME supports several phylogenetic diversity metrics, including Faith's Phylogenetic Diversity and weighted and unweighted UniFrac. In addition to counts of features per sample (i.e., the data in the ``FeatureTable[Frequency]`` QIIME 2 artifact), these metrics require a rooted phylogenetic tree relating the features to one another. This information will be stored in a ``Phylogeny[Rooted]`` QIIME 2 artifact. To generate a phylogenetic tree we will use ``align-to-tree-mafft-fasttree`` pipeline from the ``q2-phylogeny`` plugin.
+
+First, the pipeline uses the ``mafft`` program to perform a multiple sequence alignment of the sequences in our ``FeatureData[Sequence]`` to create a ``FeatureData[AlignedSequence]`` QIIME 2 artifact.
+Next, the pipeline masks (or filters) the alignment to remove positions that are highly variable. These positions are generally considered to add noise to a resulting phylogenetic tree.
+Following that, the pipeline applies FastTree to generate a phylogenetic tree from the masked alignment.
+The FastTree program creates an unrooted tree, so in the final step in this section midpoint rooting is applied to place the root of the tree at the midpoint of the longest tip-to-tip distance in the unrooted tree.
+
+.. command-block::
+
+   qiime phylogeny align-to-tree-mafft-fasttree \
+     --i-sequences rep-seqs.qza \
+     --o-alignment aligned-rep-seqs.qza \
+     --o-masked-alignment masked-aligned-rep-seqs.qza \
+     --o-tree unrooted-tree.qza \
+     --o-rooted-tree rooted-tree.qza
+
+
+.. _`fmt cdiff diversity`:
+
+Alpha and beta diversity analysis
+---------------------------------
+
+QIIME 2's diversity analyses are available through the ``q2-diversity`` plugin, which supports computing alpha and beta diversity metrics, applying related statistical tests, and generating interactive visualizations. We'll first apply the ``core-metrics-phylogenetic`` method, which rarefies a ``FeatureTable[Frequency]`` to a user-specified depth, computes several alpha and beta diversity metrics, and generates principle coordinates analysis (PCoA) plots using Emperor for each of the beta diversity metrics. The metrics computed by default are:
+
+* Alpha diversity
+
+  * Shannon's diversity index (a quantitative measure of community richness)
```

Comment:
for each metric maybe link to wikipedia or some other reference for the equations/more details on these metrics

Reply:
I think linking to a specific reference would be better than a comprehensive list, but either is fine. Nothing comes up at that link for me.

Reply:
BTW there was a power outage in the SDSC machine room so all our systems are down, thus nothing comes up in the link but should soon ... at least we hope! 😟

Reply:
k, just added those specific ones. Note that I couldn't find one for  Observed OTUs